extensions/amp-story/0.1/related-articles.js

```diff
@@ -56,11 +56,20 @@ function buildArticleFromJson_(articleJson) {
 
   user().assert(isProtocolValid(articleJson['url']),
       `Unsupported protocol for article URL ${articleJson['url']}`);
+  dev().assert(articleJson['url']);
+
+  let domain;
+  try {
+    domain = getSourceOrigin(articleJson['url']);
```

Comment:
What's the error thrown here? As it's not obvious (no error is thrown from `getSourceOrigin`, could you please add a comment?

Reply:
Done.